app/src/main/java/org/simple/clinic/contactpatient/ContactPatientUiRenderer.kt

```diff
@@ -85,6 +91,12 @@ class ContactPatientUiRenderer(
     ui.switchToCallPatientView()
   }
 
+  private fun renderPatientFacilityLabel(patientRegisteredFacilityID: UUID, currentFacility: Facility) {
+    if (patientRegisteredFacilityID == currentFacility.uuid) {
```

Comment:
Here we should compare assigned facility no?

Reply:
We can also move this condition as computed variable in the model, considering we have both facility and overdue appointment there.

Reply:
A patient will only be shown in a facility when they are assigned to that facility, right? So the assigned facility and current user facility will never be different.. Also we load overdue appointments based on the current user facility..

Reply:
So a patient will never be in the overdue appointment list if their assigned facility is not the current user facility. Also we are displaying the registered facility in the text view.

Reply:
Good point, let's go with registered facility. I have left comment for Mahima to update PRD as well. Let's move this as a computed property then.